scikeras/wrappers.py

```diff
@@ -697,20 +704,23 @@ def fit(self, X, y, sample_weight=None, **kwargs) -> "BaseWrapper":
             If not provided, then each sample is given unit weight.
         **kwargs : Dict[str, Any]
             Extra arguments to route to ``Model.fit``.
-            This functionality has been deprecated, and will be removed in SciKeras 1.0.0.
-            These parameters can also be specified by prefixing `fit__` to a parameter at initialization;
-            e.g, `BaseWrapper(..., fit__batch_size=32, predict__batch_size=1000)`.
+
+        Warnings
+        --------
+            Passing estimator parameters as keyword arguments (aka as ``**kwargs``) to ``fit`` is not supported by the Scikit-Learn API,
+            and may be deprecated in SciKeras.
```

Comment:
```suggestion
            and will be deprecated in SciKeras.
```

A ways down the road, I think this behavior will be deprecated – probably a year or two after TF drops support for Scikit-learn wrappers.

I don't think warnings should be issued for behavior that "may be deprecated."